### PR TITLE
Feature/scroll

### LINKED ISSUE #65: [Home page] The “Space2Study” logo button does not return to the home page welcoming block after clicking it.
**Description**: The “Space2Study” logo button does not return to the home page welcoming block after clicking it.

**Environment**: Windows 10 x64, Microsoft Edge 112.0.1722.48, Google Chrome 112.0.5615.122, MacOS Big Sur 11.7.3 Safari 16.3 (16614.4.6.11.6, 16614)

**Reproducible**: always.

**Preconditions:**
The client site is opened.
The user is not logged in.

**Steps to reproduce:**
1. Scroll down to the “How it works” block on the home page.
2. Click on the “Space2Study” logo button on the header.
3. Take a look at the home page.

**Actual result:**
The “Space2Study” logo button does **NOT** return to the home page welcoming block after clicking it.
**Expected result:**
The system scrolls back to the home page welcoming block after clicking on the “Space2Study” logo button on the header.

**Attachments:**

https://user-images.githubusercontent.com/129896611/233662817-59985b87-5e26-4970-9039-76b8419d3ce8.mov

### PR DESCRIPTION
**Pull request by LeonidShamarin**
https://github.com/Space2Study-Project/Space2Study-FrontEnd/assets/122298176/6f9a4a15-70ad-4416-a8d4-bb6e9d1727a5

Fixed, now when you press the button (Space2study), page  scroll to the top

### CODE REVIEW

**Review by AndriiNasadiuk — CHANGES_REQUESTED** [review]
needs test coverage >= 80%

File: .vscode/settings.json
Comment (AndriiNasadiuk) [inline .vscode/settings.json:1]: needs to explain

File: package-lock.json
Comment (AndriiNasadiuk) [inline package-lock.json:1]: 13518-13521 rows needs to explain

**Review by AndriiNasadiuk — COMMENTED** [review]
it seems too complicated. Needs explanations on meeting

**Comment by LeonidShamarin** [comment]
Another pull request make

**Pull request closed without merging**